.gitignore

```diff
@@ -48,40 +26,20 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 
-# Translations
-*.mo
-*.pot
-
-# Django stuff:
-*.log
-local_settings.py
-
-# Flask stuff:
-instance/
-.webassets-cache
-
-# Scrapy stuff:
-.scrapy
-
-# Sphinx documentation
-docs/_build/
-
-# PyBuilder
-target/
-
-# Jupyter Notebook
-.ipynb_checkpoints
+# Also ignore archive artifacts (built via "mix archive.build").
+*.ez
 
-# pyenv
-.python-version
+# Ignore package tarball (built via "mix hex.build").
+pydis_site-*.tar
 
-# celery beat schedule file
-celerybeat-schedule
+# Ignore assets that are produced by build tools.
+/priv/static/assets/
 
-# SageMath parsed files
-*.sage.py
+# In case you use Node.js/npm, you want to ignore these.
+npm-debug.log
+/assets/node_modules/
 
-# dotenv
+# Ignore custom local environment variables.
 .env
 
```

Comment:
If we're moving towards making the site blazingly fast, we need to put in some safeguards.
```suggestion
# Ignore unwanted files
*.rs
```